### DISCUSSION TITLE
how to disable collection loading when execute bash standalone_embed.sh start？

### DISCUSSION THREAD

**Discussion by kksasa** [milvus-io/milvus#36756] [Q&A and General discussion]
standalone milvus with docker cannot start now. Doubt data was broken, so how to disable collection loading when start milvus?

**Comment by liliu-z** [comment]
Use [`birdwatcher`](https://github.com/milvus-io/birdwatcher) to force delete collection.
Steps:
1. connect `etcd` with birdwatcher
2. run `force-release`

**Reply by kksasa** [reply]
why so complicated, unfortunately, my linux server cannot access external web server.

I just need not load collection during run bash standalone_embed.sh start , any way to do it?

**Reply by xiaofan-luan** [reply]
@kateshaowanjou could you collect logs with docker log so we know what cause the problem.

There is nothing we can do withou more detailed info.
